pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -191,6 +198,18 @@ export class Terminal extends React.Component {
     render() {
         return (
             <>
+                <Modal title={_("Pasting in cockpit terminal")}
+                       position="top"
+                       variant="small"
+                       isOpen={this.state.showPastingModal}
+                       onClose={() => this.setState({ showPastingModal: false })}
+                       actions={[
+                           <Button key="cancel" variant="secondary" onClick={() => this.setState({ showPastingModal: false })}>
+                               {_("Close")}
+                           </Button>
+                       ]}>
+                    {_("This action is unavailable using the context menu, but you can still use Shift+Insert for paste.")}
```

Comment:
Like this it sounds a bit confusing, like we papered over our own bug. Maybe "Your browser does not allow paste from the context menu. You can use Shift+Insert."

Reply:
I like the suggestion, replaced.